src/upstream/teleEmitter.ts

```diff
@@ -67,6 +67,17 @@ export class TeleReporterEmitter implements ReporterV2 {
     });
   }
 
+  onTestError(test: reporterTypes.TestCase, result: reporterTypes.TestResult, error: reporterTypes.TestError): void {
```

Comment:
Shouldn't there be an empty `errors` array somewhere in `onTestEnd`? Otherwise we emit errors twice.